### PR TITLE
creating-adaptive-xl

### PR DESCRIPTION
**Pull request by Konmly**
Сreating an adaptive xl layout.

localhost_3000_

### CODE REVIEW

File: src/scss/_footer.scss [hunk 1]
```diff
@@ -18,17 +18,31 @@
...
     }
+
+    @media screen and (max-width: $xl) {
+      column-gap: 64px;
+    }
   }
 
   &__tariff {
-    @include flex($direction: column);
+    @include flex($direction: column, $align: stretch);
```
Comment (PahaTonk) [inline src/scss/_footer.scss:45]: Have you tested it on previous screens?

File: src/scss/_footer.scss [hunk 2]
```diff
@@ -41,19 +55,35 @@
       padding: 50px 65px 45px;
 
       background-image: url('../images/background1.png');
+
+      @media screen and (max-width: $xl) {
```
Comment (PahaTonk) [inline src/scss/_footer.scss:59]: this is for `&__tariff`, please move it after line 52

**Review by PahaTonk — CHANGES_REQUESTED** [review]

File: src/scss/_footer.scss (hunk 1 above)
Comment (Konmly) [reply src/scss/_footer.scss:45]: Yes, it's fine

File: src/scss/_footer.scss (hunk 2 above)
Comment (Konmly) [reply src/scss/_footer.scss:59]: q1

**Review by PahaTonk — APPROVED** [review]

**Pull request merged**